### ISSUE TITLE
krill and krillc do not use -c consistently. this is confusing

### ISSUE THREAD

**Issue by geeohgeegeeoh** [NLnetLabs/krill#213]
krill -c conf.file
krillc -c CA instance

this is deeply confusing. krillc should be able to be told a .conf file and read it, or the krillc -h needs work to explain.

**Comment by timbru** [comment]
Fair point `-c` is allowed as a short hand for `--ca`, but I don't think it's necessary - the latter is not very long, and it's good to be explicit.

There is currently no way to point krillc at a krill.conf file, but this can be added in future. A krill.conf file contains the information that krillc needs to talk to a server (server uri and token), so reading this can be one more alternative to supplying these values as command line arguments or ENV variables.

I think this will have to wait until the version after 0.5.1 though, because of time constraints.
